### PR TITLE
Changes for django 1.10 + installation instructions.

### PR DESCRIPTION
**Pull request by DrDub**
Instructions to run a dev version. Tested on python2.7.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,11 @@ Coming soon:
 - tests for our code!
 - feedback box for more questions and other suggestions
 - moderated discussion page(s) 
+
+
+## Installation instructions (development)
+
+- edit django_project/sample_settings.py into settings.py
```
Comment (evykassirer) [inline README.md]: "rename `django_project/sample_settings.py` to `django_project/settings.py`" would be more clear I think

File: README.md [hunk 2]
```diff
@@ -14,3 +14,11 @@ Coming soon:
...
+- python manage.py migrate --run-syncdb
+- python manage.py loaddata data.json
+- python manage.py runserver
```
Comment (evykassirer) [inline README.md]: then run these commands:
```
python manage.py migrate --run-syncdb
python manage.py loaddata data.json
python manage.py runserver
```

(the backticks make it a bit easier to see at a glance that you're running a command)

File: README.md [hunk 3]
```diff
@@ -14,3 +14,11 @@ Coming soon:
 - tests for our code!
 - feedback box for more questions and other suggestions
 - moderated discussion page(s) 
+
+
+## Installation instructions (development)
+
```
Comment (evykassirer) [inline README.md:20]: can you add an instruction to install the right django version? that was the one bump I encountered when trying to get it run locally

**Review by evykassirer — CHANGES_REQUESTED** [review]
love this! thanks for adding this - just a few nit comments

I'd love to know - do you plan on using this for something?

**Review by evykassirer — APPROVED** [review]

**Comment by evykassirer** [comment]
thanks!

**Comment by DrDub** [comment]
Answering your question, I do not plan to use this at the moment, but it seems a nice project and worth improving.

**Pull request merged**